project/Build.scala

```diff
@@ -774,6 +774,19 @@ object Build {
           previousArtifactSetting,
           mimaBinaryIssueFilters ++= BinaryIncompatibilities.SbtPlugin,
 
+          /* Work around https://github.com/sbt/sbt/issues/3393.
+           * Should be:
+           *   addSbtPlugin("org.portable-scala" % "sbt-platform-deps" % "1.0.0")
+           * We inline the fixed definition of addSbtPlugin to be
+           * released with sbt 0.13.17.
+           */
```

Comment:
Shouldn't we just upgrade?

Reply:
We would lose the guarantee that sbt-scalajs can be used with sbt 0.13.16.

But maybe that's fine?

Reply:
Oh, I thought we have upgraded patch sbt versions at will in the past... If we haven't, I'm fine with either and leave it up to you.